src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -383,3 +383,31 @@ method requestMoreMessages*(self: Module) =
 
 method fillGaps*(self: Module, messageId: string) =
   self.controller.fillGaps(messageId)
+
+method joinGroupChat*(self: Module) =
+  self.controller.joinGroupChat()
+
+method leaveChat*(self: Module) =
+  self.controller.leaveChat()
+
+method didIJoinedChat*(self: Module): bool =
```

Comment:
This has an English error.
It can be changed to `haveIJoinedChat` or `isChatJoined`